### PR TITLE
Feature/youtube channel template

### PR DESCRIPTION
**Pull request by Escoly**

### CODE REVIEW

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title>Paco's Channel</title>
+</head>
+
+<body>
+
+    <header>
+        <section>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: Hi @Escoly ! This `section` tag is not necessary here

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                <button><i class="searchbar-sidebuttons fas fa-search"></i></button>
+                <button><i class="searchbar-sidebuttons fas fa-ellipsis-v"></i></button>
+
+            </section>
+    </header>
+
+    <div class="content">
+        <div class="side-navbar">
+            <div class="side-navbar-top-section">
+                <p>Inicio</p>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: Here better use `a` tag inside the `nav` tag. For example:

```
<nav class="side-navbar-top-section">
  <ul>
    <li><a href="#" title="Inicio">Inicio</a></li>
    <li><a href="#" title="Tendencias">Tendencias</a></li>
    <li><a href="#" title="Historial">Historial</a></li>
  </ul>
</nav>
```

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                <p>Tendencias</p>
+                <p>Historial</p>
+            </div>
+
+            <div class="side-navbar-middle-section">
+                <h4>LO MEJOR DE YOUTUBE</h4>
+                <p>Música</p>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: The same comment that [here](https://github.com/manuelbarzi/skylab-bootcamp-201904/pull/100/files#r272976927)

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+                <p>Deportes</p>
+                <p>Videjuegos</p>
+                <p>Peliculas</p>
+                <p>Noticias</p>
+                <p>En Directo</p>
+                <p>Video en 360º</p>
+            </div>
+            <div class="side-navbar-explore">
+                <p>Explorar canales</p>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: The same comment that [here](https://github.com/manuelbarzi/skylab-bootcamp-201904/pull/100/files#r272976927)

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+            </div>
+            <div class="side-navbar-more">
+                <h4>MAS DE YOUTUBE</h4>
+                <p>Youtube Premium</p>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: The same comment that [here](https://github.com/manuelbarzi/skylab-bootcamp-201904/pull/100/files#r272976927)

File: staff/albert-escolar/youtube-channel-template/index.html [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+                <p>Directo</p>
+            </div>
+            <div class="side-navbar-settings">
+                <p>Configuración</p>
```
Comment (robert-z) [inline staff/albert-escolar/youtube-channel-template/index.html]: The same comment that [here](https://github.com/manuelbarzi/skylab-bootcamp-201904/pull/100/files#r272976927)

File: staff/albert-escolar/youtube-channel-template/index.html (hunk 1 above)
Comment (Escoly) [reply staff/albert-escolar/youtube-channel-template/index.html]: Fixed!

**Pull request merged**